global_config/afqts_domain.sh

```diff
@@ -0,0 +1,10 @@
+CONFIG=production
+CONFIG_SHORT=pd
+AZURE_SUBSCRIPTION=s189-teacher-services-cloud-production
+RESOURCE_NAME_PREFIX=s189p01
+ENV_TAG=Prod
+DNS_ZONE=afqts
+RESOURCE_GROUP_NAME=${RESOURCE_NAME_PREFIX}-${DNS_ZONE}domains-rg
+KEYVAULT_NAME=${RESOURCE_NAME_PREFIX}-${DNS_ZONE}domains-kv
+STORAGE_ACCOUNT_NAME=${RESOURCE_NAME_PREFIX}${DNS_ZONE}domainstf
+DOMAINS_ID=afqts
```

Comment:
This looks very similar to the production global configuration we're going to add for AKS (https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1433/files) but with some additional fields. Can we use the same config?

Especially the variables `DOMAINS_ID` and `DNS_ZONE`, these would never change so we can define them as static variables in the `Makefile`?

Reply:
Reviewed with Thomas, Will clean up in upcoming PR's